docs/data/date-pickers/date-time-picker/CustomDateTimePicker.tsx

```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
 import dayjs from 'dayjs';
+import { DemoContainer } from 'docsx/src/modules/components/DemoContainer';
 import AlarmIcon from '@mui/icons-material/Alarm';
 import SnoozeIcon from '@mui/icons-material/Snooze';
 import ClockIcon from '@mui/icons-material/AccessTime';
 import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import { Unstable_NextDateTimePicker as NextDateTimePicker } from '@mui/x-date-pickers/NextDateTimePicker';
-import Stack from '@mui/material/Stack';
 
 export default function CustomDateTimePicker() {
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <Stack spacing={3}>
+      <DemoContainer>
```

Comment:
The demos are once again inline after changes and IMHO could use a bit more width to account for the full format placeholder
![Screenshot 2022-12-09 at 15 14 31](https://user-images.githubusercontent.com/4941090/206710489-7d6fe298-23f2-4177-84d9-2b7a8fa9729b.png)

Reply:
See https://github.com/mui/mui-x/pull/7149#discussion_r1045490074